lib/taskana-core/src/test/java/acceptance/workbasket/QueryWorkbasketAccTest.java

```diff
@@ -94,14 +95,14 @@ public void testQueryAllForAdminMultipleTimes() {
     public void testQueryWorkbasketValuesForColumnName() {
         WorkbasketService workbasketService = taskanaEngine.getWorkbasketService();
         List<String> columnValueList = workbasketService.createWorkbasketQuery()
-            .listValues("NAME", null);
+            .listValues(WorkbasketQueryColumnName.NAME, SortDirection.ASCENDING);
```

Comment:
may be keep the default by using null

Reply:
Yes, this was a little oversight by me, I had it changed at an earlier stage, but now it is back how it was.